{"schemaVersion":{"major":0,"minor":3,"patch":0},"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["iOS"],"tokens":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"languages":["occ"]}]}],"hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"]]},"identifier":{"interfaceLanguage":"occ","url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed"},"kind":"symbol","abstract":[{"type":"text","text":"The passed in token was malformed."}],"sections":[],"metadata":{"fragments":[{"kind":"identifier","text":"SMIChallengeReasonMalformed"}],"title":"SMIChallengeReasonMalformed","symbolKind":"case","modules":[{"name":"SMIClientCore"}],"externalID":"c:@E@SMIChallengeReason@SMIChallengeReasonMalformed","navigatorTitle":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}],"role":"symbol","roleHeading":"Enumeration Case"},"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed"]}],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"type":"topic","role":"collection","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","title":"SMIClientCore","url":"\/documentation\/smiclientcore"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason":{"fragments":[{"text":"SMIChallengeReason","kind":"identifier"}],"kind":"symbol","role":"symbol","url":"\/documentation\/smiclientcore\/smichallengereason","type":"topic","title":"SMIChallengeReason","abstract":[{"text":"The reason for a credential challenge. Possible values:","type":"text"}],"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIChallengeReason/SMIChallengeReasonMalformed":{"abstract":[{"type":"text","text":"The passed in token was malformed."}],"kind":"symbol","type":"topic","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIChallengeReason\/SMIChallengeReasonMalformed","title":"SMIChallengeReasonMalformed","url":"\/documentation\/smiclientcore\/smichallengereason\/smichallengereasonmalformed","role":"symbol","fragments":[{"text":"SMIChallengeReasonMalformed","kind":"identifier"}]}}}